services/metadata/frontend/project-page/DefaultTabComponent.svelte

```diff
@@ -0,0 +1,203 @@
+    {#if Array.isArray(dataset?.content.abstract)}
+    <div id=abstract class="data {isAbstractExpanded ? '' : 'abstract-short'}">
+      {#each dataset?.content.abstract as a}
+        {#if a.url}
+        <a class=data href={a.url} target=_>{a.name}</a>
+        {:else}
+        <span>{a}</span>
+        {/if}
+      {/each}
+    </div>
+    {/if}
+  </div>
+
+  {#if abstractLinesNumber >= 6}
+  <div on:click={toggleExpand} class=expand-button>show {isAbstractExpanded ? "less" : "more"}</div>
+  {/if}
+
+  <span class=label>Attributions</span>
+  <div class="grid-wrapper">
+    {#if Array.isArray(dataset?.content.qualifiedAttribution)}
+      {#each dataset?.content.qualifiedAttribution as a}
+      <div class="attribution">
+        <div style="color: olivedrab">{a.role}</div>
+        {#if findObjectById(a.agent[0].id).type === "http://ns.dasch.swiss/repository#Person"}
+        <div>{findObjectById(a.agent[0].id)?.givenName.split(";").join(" ")} {findObjectById(a.agent[0].id)?.familyName.split(";").join(" ")}</div>
+        {#if findObjectById(a.agent[0].id)?.sameAs}
+        <a href={findObjectById(a.agent[0].id)?.sameAs[0].name} target=_>{findObjectById(a.agent[0].id)?.sameAs[0].name}</a>
+        {/if}
+        {#if findObjectById(a.agent[0].id)?.email}
+        <div>{findObjectById(a.agent[0].id)?.email[0]}</div>
+        {/if}
+        {#if Array.isArray(findObjectById(a.agent[0].id)?.memberOf)}
+          {#each findObjectById(a.agent[0].id)?.memberOf as o}
+          <div>{findObjectById(o.id).name}</div>
+          {/each}
+        {/if}
+        <div>{findObjectById(a.agent[0].id)?.jobTitle[0]}</div>
+        {:else}
+        <div>{findObjectById(a.agent[0].id)?.name}</div>
+        {/if}
+        <br />
+      </div>
+      {/each}
+    {/if}
+  </div>
+
+  {/if}
+</div>
+
+<style>
+  a {
+    color: var(--dasch-violet);
+  }
+  .property-row {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+  }
+  .label, .data {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    flex: 2;
+    margin-bottom: 10px;
+    word-break: break-word;
+    width: fit-content;
+  }
+  .label {
+    margin: 10px 0;
+    flex: 1;
+    font-weight: bold;
+  }
+  .abstract-short {
+    display: -webkit-box;
+    -webkit-line-clamp: 6;
+    -webkit-box-orient: vertical;
+    overflow: hidden;
+    height: 45x;
+    line-height: 18px;
+  }
+  .expand-button {
+    background-image: linear-gradient(to right, #fff, var(--dasch-grey-3), #fff);
+    color: var(--dasch-violet);
+    text-align: center;
+    font-size: 0.8em;
+    padding: 2px 0;
+    cursor: pointer;
+  }
+  .grid-wrapper {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+  }
+  @media screen and (min-width: 576px) {
+    .grid-wrapper {
+      grid-template-columns: repeat(2, 1fr);
+    }
+  }
+  @media screen and (min-width: 768px) {
+    .grid-wrapper {
+      grid-template-columns: repeat(3, 1fr);
+    }
+  }
+</style>
```

Comment:
we should probably at some point decide on how big we want our components to be at max; and when we want to start splitting things up into smaller components.  
(Not saying this is too big, just made me think.)

Reply:
I see two main reasons for splitting:
- functional - better reusability + less code redundancy
- structural - if it grows to big, it's harder to maintain and read the code

With the complete design provided upfront it's easier to predict how to divide the app into components. Here w'll need to adapt this strategy in further iterations.